packages/wonder-blocks-popover/src/components/popover-content-core.tsx

```diff
@@ -70,6 +70,7 @@ export default class PopoverContentCore extends React.Component<Props> {
 
     render(): React.ReactNode {
         const {
+            "aria-label": ariaLabel,
```

Comment:
Oh interesting, so the ariaLabel prop is exposed, but the value is not piped through to View component??

Reply:
Yup it accepts all the `aria-*` labels by default, but we have to expose them individually @catandthemachines